modules/user_admin/show_users.php

```diff
@@ -55,22 +55,40 @@ function exec_ogp_module() {
 	if(hasValue($loggedInUserInfo) && is_array($loggedInUserInfo) && $loggedInUserInfo["users_page_limit"] && !hasValue($_GET['limit'])){
 		$limit_user = $loggedInUserInfo["users_page_limit"];
 	}
-	
-    echo '<h2>'.get_lang('users')."</h2>";
-	echo '<form action="home.php" method="GET" style="float:left;">
-		<p><a href="?m=user_admin&amp;p=add">'.get_lang("add_new_user").'</a></p>
-		<input type ="hidden" name="m" value="user_admin" />
-		<input name="search" type="text" id="search" />
-		<input type="submit" value="'.get_lang('search').'" />
-		</form>';
+
+	echo '<h2>'.get_lang('users')."</h2>";
+
+	$result = $db->getUserList_limit($page_user, $limit_user, $search_field);
+
+	if (empty($result) && $search_field !== false) {
+		print_failure(get_lang_f('no_results_found', htmlentities($search_field)));
+
+		$view->refresh("?m=user_admin", 5);
+		return;
+	}
+
+	echo '<table style="width: 100%;">
+			<tr>
+				<td style="width: 50%; vertical-align: middle; text-align: left;">
+					<p><a href="?m=user_admin&amp;p=add">'.get_lang("add_new_user").'</a></p>
+				</td>
+				<td style="width: 50%; vertical-align: middle; text-align: right;">
+					<form action="home.php" method="GET" style="float:right;">
+					<input type ="hidden" name="m" value="user_admin" />
+					<input name="search" type="text" id="search" />
+					<input type="submit" value="'.get_lang('search').'" />
+					</form>
+				</td>
+			</tr>
+		</table>';
+
     echo '<table class="userListTable center" style="width: 100%;margin-top:100px;">';
```

Comment:
would change the margin top line 85 to 50px now everything is not on top of each other, and that would match the margin bottom for the show_homes page.